### PR TITLE
fix: rewrite playwright logs from errors

### LINKED ISSUE #88: stacktrace gets really huge when step fails on finding DOM element
+ When hovering/clicking over certain elements. The playwright API automatically retries to find that particular DOM element, when the step fails this creates a huge stack trace which is super hard to read. 

This is a bug and we should only show/print the relevant error and hide other stuffs. 

### Example stack trace
```
Journey: short
   ✓  Step: 'load homepage' succeeded (3585 ms)
^C   ✖  Step: 'hover over products menu' failed (348 ms)
   ---
      stack: |-
        Error: page.hover: Frame has been detached.
        =========================== logs ===========================
        [api] waiting for selector "css=[data-nav-item=products]"
        [api]   selector resolved to visible <li data-nav-item="products" class="jsx-4008395266 n…>…</li>
        [api] attempting hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        [api]   done scrolling
        [api]   checking that element receives pointer events at (267.02,42)
        [api]   element does not receive pointer events
        [api] retrying hover action
        [api]   waiting for element to be visible, enabled and not moving
        [api]   element is visible, enabled and does not move
        [api]   scrolling into view if needed
        ============================================================
        Note: use DEBUG=pw:api environment variable and rerun to capture Playwright logs.
            at CRPage._sessionForFrame (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/chromium/crPage.js:75:23)
            at CRPage._sessionForHandle (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/chromium/crPage.js:82:21)
            at CRPage.scrollRectIntoViewIfNeeded (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/chromium/crPage.js:201:21)
            at ElementHandle._scrollRectIntoViewIfNeeded (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/dom.js:167:43)
            at ElementHandle._performPointerAction (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/dom.js:279:37)
            at ElementHandle._retryPointerAction (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/dom.js:242:28)
            at /Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/frames.js:674:32
            at ProgressController.run (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/progress.js:75:28)
            at Frame.hover (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/frames.js:731:9)
            at Step.callback (/Users/vigneshh/elastic/synthetic-monitoring/examples/suites/journey2.js:8:5)
          -- ASYNC --
            at Frame.hover (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/helper.js:79:23)
            at /Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/page.js:401:61
            at Page._attributeToPage (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/page.js:370:20)
            at Page.hover (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/page.js:401:21)
            at Page.hover (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/helper.js:80:31)
            at Step.callback (/Users/vigneshh/elastic/synthetic-monitoring/examples/suites/journey2.js:8:16)
            at Runner.runStep (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:126:18)
            at Runner.runSteps (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:159:27)
            at Runner.runJourney (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:215:27)
            at Runner.run (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:257:29)
          -- ASYNC --
            at Page.hover (/Users/vigneshh/elastic/synthetic-monitoring/node_modules/playwright/lib/helper.js:79:23)
            at Step.callback (/Users/vigneshh/elastic/synthetic-monitoring/examples/suites/journey2.js:8:16)
            at Runner.runStep (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:126:18)
            at Runner.runSteps (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:159:27)
            at Runner.runJourney (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:215:27)
            at Runner.run (/Users/vigneshh/elastic/synthetic-monitoring/src/core/runner.ts:257:29)
            at Object.run (/Users/vigneshh/elastic/synthetic-monitoring/src/index.ts:24:5)
            at exec (/Users/vigneshh/elastic/synthetic-monitoring/src/cli.ts:107:3)
            at cli (/Users/vigneshh/elastic/synthetic-monitoring/src/cli.ts:124:5)
   ---



```

### PR DESCRIPTION
**Pull request by vigneshshanmugam**
+ fix #88 
+ Introduce a rewrite stack function that rewrites the playwright specific logs message and strips them to only two adjacent  lines to keep the error meaningful instead of giving repetitive lines on every retries. 
+ Also removed the message field from error as it contains the same repetetive message from the error stack. UI can only show stacktrace and skip showing errors.

### CODE REVIEW

File: src/cli.ts [hunk 1]
```diff
@@ -41,9 +41,12 @@ import { run } from './';
 const program = parseArgs();
 const resolvedCwd = cwd();
 /**
- * Set debug based on flag
+ * Set debug based on DEBUG ENV and -d flags
+ * namespsace - es - elastic synthetics
```
Comment (andrewvc) [inline src/cli.ts:45]: `s/namespsace/namespace/`

Maybe we should make the namespace `synthetics`? `es` sounds like Elasticsearch for me, and is more common within our org. Maybe `synthetics` is more appropriate? Longer, I know.

Also, can we document this somewhere? This could be a separate PR as well.

Comment (vigneshshanmugam) [reply src/cli.ts:45]: Ahh, i was going for `Elastic synthetics` as `es` and didnt even think of Elasticsearch :D 

Yeah `synthetics` or `syn` for short works as well?

File: src/helpers.ts [hunk 2]
```diff
@@ -49,8 +49,8 @@ export function formatError(error: Error) {
   if (!(error instanceof Error)) {
     return;
   }
-  const { name, message, stack } = error;
-  return { name, message, stack };
```
Comment (vigneshshanmugam) [inline src/helpers.ts:53]: error.message has to be rewritten as well as its required in the UI.

**Comment by vigneshshanmugam** [comment]
@andrewvc I opened #122 to address the same issue, I deleted my fork my mistake and cannot able to get my branch back 🤷‍♂️ 
Sorry for moving that.
